pom.xml

```
@@ -3,40 +3,41 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
+    <version>3.43</version>
+    <relativePath />
   </parent>
 
   <artifactId>form-element-path</artifactId>
   <version>1.9-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Form element path plugin</name>
   <description>Introduce unique path attributes for UI controls to facilitate Jenkins UI testing.</description>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/Form+Element+Path+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/Form+Element+Path+Plugin</url>
 
   <properties>
-    <jenkins.version>1.580</jenkins.version>
-    <java.level>6</java.level>
+    <jenkins.version>2.176.1</jenkins.version>
```

would be helpful to also use it internally for our older proprietary fixed line?
Doesn't make too much difference IIUC as that does not have the installation of detached plugins for existing installed plugins available.

```suggestion
    <jenkins.version>2.138.1</jenkins.version>
```

---

> the installation of detached plugins for existing installed plugins

https://github.com/jenkinsci/jenkins/pull/4039 for the record.

---

And…that baseline would work on Java 8 but not 11, as per https://github.com/jenkinsci/form-element-path-plugin/pull/8#discussion_r292442826.